packages/kotti-ui/source/kotti-breadcrumb/KtBreadcrumb.vue

```diff
@@ -25,11 +25,34 @@
 	</div>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api'
+import { isBoolean, isString } from 'lodash'
+
+import { isYocoIcon } from '../validators'
+
+import { KottiBreadcrumb } from './types'
+
+const breadcrumbValidator = (
+	value: KottiBreadcrumb.Breadcrumb,
+): value is KottiBreadcrumb.Breadcrumb =>
+	value !== null &&
+	typeof value === 'object' &&
+	isBoolean(value.isCompleted) &&
+	isString(value.title) &&
+	isString(value.to)
+
+export default defineComponent<KottiBreadcrumb.PropsInternal>({
 	name: 'KtBreadcrumb',
 	props: {
-		breadcrumbs: { type: Array, required: true },
+		breadcrumbs: {
+			type: Array,
+			required: true,
+			validator: (value: unknown) =>
+				Array.isArray(value) &&
+				value.length > 0 &&
```

Comment:
would you rather error on prop validation if `breadcrumbs` is an empty array at one point in time (perhaps before mounting a component in the user-app) or just not loading the breadcrumbs if that's the case ? b/c in the latter, this line wouldn't be needed, and instead the `v-for` in the template would render nothing